analytics_data_api/v0/tests/views/test_course_summaries.py

```diff
@@ -103,13 +108,20 @@ def expected_result(self, item_id, modes=None, availability='Current', programs=
             summary['programs'] = [CourseSamples.program_ids[0]]
         return summary
 
-    def all_expected_results(self, ids=None, modes=None, availability='Current', programs=False):  # pylint: disable=arguments-differ
+    def all_expected_results(self,  # pylint: disable=arguments-differ
+                             ids=None,
+                             modes=None,
+                             availability='Current',
+                             programs=False,
+                             recent_count_change=None):
         if modes is None:
             modes = enrollment_modes.ALL
 
-        return super(CourseSummariesViewTests, self).all_expected_results(ids=ids, modes=modes,
-                                                                          availability=availability,
-                                                                          programs=programs)
+        return super(CourseSummariesViewTests, self)\
```

Comment:
I don't like backslashes, so this is just a personal preference recommendation, but, how do you think this looks:

```
        return super(CourseSummariesViewTests, self).all_expected_results(
            ids=ids,
            modes=modes,
            availability=availability,
            programs=programs,
            recent_count_change=recent_count_change
        )
```

Reply:
I like that better, I'll see what pylint thinks of that, butit didn't like the variant of this that I tried.  There's a lot of things I disagree with pylint/pep8 about.